cmake/modules/RootMacros.cmake

```diff
@@ -1179,18 +1188,19 @@ function(ROOT_STANDARD_LIBRARY_PACKAGE libname)
   endif()
 
   ROOT_GENERATE_DICTIONARY(G__${libname} ${ARG_HEADERS}
-                          ${MODULE_GEN_ARG}
+                          ${NO_CXXMODULE_FLAG}
                           ${STAGE1_FLAG}
+                          MODULE ${libname}
                           LINKDEF ${ARG_LINKDEF}
                           NODEPHEADERS ${ARG_NODEPHEADERS}
                           OPTIONS ${ARG_DICTIONARY_OPTIONS}
                           DEPENDENCIES ${ARG_DEPENDENCIES}
                           BUILTINS ${ARG_BUILTINS}
                           )
 
-  if(ARG_NO_MODULE)
-    target_sources(${libname} PRIVATE G__${libname}.cxx)
-  endif()
+#  if(ARG_NO_MODULE)
+#    target_sources(${libname} PRIVATE G__${libname}.cxx)
+#  endif()
```

Comment:
Why is this disabled? We need to add the source for the dictionary to the target library one way or another.

Reply:
We always pass the `module` option. The old `NO_MODULE` was a little misused. It occurs in one place in ROOT and it means "do not build a c++ module for this particular library". This forced us to complicate the `ROOT_GENERATE_DICTIONARY` to be conditional to `ARG_MODULE` when we actually meant to disable the C++ modules for the particular library.

Reply:
Sorry, I didn't notice the addition of `MODULE ${libname}` unconditionally above. In that case this is fine, but please actually remove the code rather than comment it out.